packages/notifications/resources/views/components/body.blade.php

```diff
@@ -1,5 +1,5 @@
 <p
-    {{ $attributes->class(['fi-no-notification-body text-sm text-gray-500 dark:text-gray-400']) }}
+    {{ $attributes->class(['fi-no-notification-body text-sm text-gray-500 dark:text-gray-400 break-all']) }}
```

Comment:
Shouldn't we use `break-words` instead?
https://tailwindcss.com/docs/word-break#break-words

Reply:
`break-all` also breaks text when it shouldn't, so we can't use that.

Reply:
Agree. This does not look nice. What about using `break-words overflow-hidden`? This solves the problem as well.

Reply:
Is `overflow-hidden` really required for it to work?

Reply:
As far as I can tell: Yes. See screenshots.

Without `overflow-hidden`:

![without_overflow_hidden](https://github.com/filamentphp/filament/assets/9416642/76795323-7759-47bf-8cc1-3430e57fc5ed)

With `overflow-hidden`:
![with_overflow_hidden](https://github.com/filamentphp/filament/assets/9416642/fafc1329-618b-425e-b8c4-dc14623bd44e)

Reply:
Alright, thanks.